types/moment-jalaali/index.d.ts

```diff
@@ -15,6 +15,12 @@ declare module 'moment' {
          * Use persian digits as decribed by unicode
          */
         usePersianDigits?: boolean;
+        /**
+         * use dialect option to change usePersian dialect, available options are:
+         *      persian: default dialect(امرداد، آدینه، ...)
+         *      persian-modern: modern dialect(مرداد، جمعه، ...)
+         */
+        dialect?: string;
```

Comment:
Change this to:
`dialect?: 'persian' | 'persian-modern';`

Reply:
Ok, Done 😊